google-cloud-clients/google-cloud-firestore/src/test/java/com/google/cloud/firestore/it/ITSystemTest.java

```diff
@@ -920,6 +920,10 @@ public void queryWatch() throws Exception {
     //        case there is a hang during the test the whole suite isn't taken with it)
     //   D. After all updates have been performed and the expected number of events have been
     //      received, assertions on the events will be performed.
+    //
+    // Note: There is a potential that this test fails if the Firestore backend chooses not send us
```

Comment:
`s/chooses not send us/chooses to not send us/`

Reply:
Fixed